src/devices/cpu/z80/z80.cpp

```diff
@@ -1461,6 +1395,23 @@ inline void z80_device::ei()
 	m_after_ei = true;
 }
 
+inline void z80_device::set_f(u8 f)
+{
+	m_qtemp = 0;
+	F = f;
+}
+
+inline void z80_device::illegal_1() {
+	LOGUNDOC("ill. opcode $%02x $%02x ($%04x)\n",
+			m_opcodes.read_byte(translate_memory_address((PCD-1)&0xffff)), m_opcodes.read_byte(translate_memory_address(PCD)), PCD-1);
+}
+
+inline void z80_device::illegal_2()
+{
+	LOGUNDOC("ill. opcode $ed $%02x\n",
+			m_opcodes.read_byte(translate_memory_address((PCD-1)&0xffff)));
+}
```

Comment:
More K&R formatting.  Also, you changed most of the file to have spaces around binary operators – why no spaces in the expressions here?